### DISCUSSION TITLE
pxrUsd Plugin fails to load in Maya 2020

### DISCUSSION THREAD

**Discussion by Akhilesh09** [Autodesk/maya-usd#1155] [Q&A] [answered]
initializePlugin function failed (pxrUsd)

**Comment by mattyjams** [comment, marked as answer]
Is there any more information available here?

During initialization, the `pxrUsd` plugin sources Mel script files and imports Python modules:
https://github.com/Autodesk/maya-usd/blob/8510159e84754f101f7b0aaa5983d1c3d8e09e6e/plugin/pxr/maya/plugin/pxrUsd/plugin.cpp#L85
https://github.com/Autodesk/maya-usd/blob/8510159e84754f101f7b0aaa5983d1c3d8e09e6e/plugin/pxr/maya/plugin/pxrUsd/plugin.cpp#L110

So this can happen if not all of your path variables have been set correctly. If you use the module file(s) included in maya-usd, this should be taken care of for you:
https://github.com/Autodesk/maya-usd/blob/dev/doc/build.md#how-to-load-plug-ins-in-maya

**Reply by kxl-adsk** [reply]
Converted to discussion for now, since there isn't anything we can act on.
